packages/contracts/test/L1/DepositFeed.spec.ts

```diff
@@ -164,18 +209,55 @@ describe('DepositFeed', () => {
         })
       })
 
+      it('when a contract deposits a transaction with an ETH value.', async () => {
```

Comment:
The dummy contract forwards ETH, as opposed to spending existing balance. During execution it's effectively the same (contract balance increases and decreases), but do we need to test this the other way?

Reply:
I think it's fine. There's zero difference in the execution context of the DepositFeed right?